README.md

```diff
@@ -42,11 +43,38 @@ to execute the unit tests:
 
 ```sh
 make test # run all tests
-cd src && cabal v2-test # run haskell tests
+cabal v2-test # run haskell tests
 ```
 
 To update the project dependencies run:
 
 ```sh
 nix flake update
 ```
+
+# Usage
```

Comment:
There is also a `--help` option for each one of them that describes all the available command line options.